.github/workflows/build.yml

```diff
@@ -0,0 +1,68 @@
+name: Build to GitHub pages
+on:
+  push:
+    branches:
+      - master 
+  pull_request:
+    branches:
+      - master 
+
+jobs: 
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      ### PREP ###
+      - name: Checkout repo
+        uses: actions/checkout@v2.3.1
+        with:
+          persist-credentials: false # needed for JamesIves/github-pages-deploy-action
+
+      ### RUBY ###
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6.3
+      - name: Cache Ruby gems
+        id: cache-gems
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-1-${{ hashFiles('Gemfile.lock') }}
+      - name: Set Ruby gems directory
+        run: bundle config path vendor/bundle
+      - name: Install Ruby gems
+        run: bundle install --jobs 4 --retry 3 --deployment
+      - name: Install node 
+        run: npm install 
```

Comment:
Do you need to install npm packages? Looking at `package.json` it was added by me 5 years ago. Likely none of it is needed.